tests/test_cassettes/s3/test_s3_tape_cassette.py

```diff
@@ -359,3 +361,110 @@ def sampling_calculator(category, size, r):
             new_cassette.save_recording(recording)
             with self.assertRaises(NoSuchRecording):
                 new_cassette.get_recording(recording.id)
+
+    def test_metadata_content_filter_by_number_value(self):
```

Comment:
You should have all these tests in the TapeCassette tests module as they are actually testing the base class, if there is no such module, please create one

Reply:
Done